src/controllerbluetooth.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+option=$1
+
+if [option = connect]
+    echo 'connect 5C:BA:37:26:6D:9A\nquit' | sudo bluetoothctl
+then
+if[option = disconnect]
```

Comment:
`if[option`

Please test your code before pushing it. This is such an easy syntax error to catch that the GitHub syntax highlighter caught it.